pkgs/os-specific/linux/power-profiles-daemon/default.nix

```diff
@@ -101,7 +102,8 @@ stdenv.mkDerivation rec {
   postPatch = ''
     patchShebangs --build \
       tests/integration-test.py \
-      tests/unittest_inspector.py
+      tests/unittest_inspector.py \
+      src/powerprofilesctl
```

Comment:
This makes it so that build python (including `dbusmock`) is used as interpreter of `$out/bin/powerprofilesctl`.

Reply:
Maybe something like

```patch
diff --git a/pkgs/os-specific/linux/power-profiles-daemon/default.nix b/pkgs/os-specific/linux/power-profiles-daemon/default.nix
index 51b1b80001b2..9e94024bf829 100644
--- a/pkgs/os-specific/linux/power-profiles-daemon/default.nix
+++ b/pkgs/os-specific/linux/power-profiles-daemon/default.nix
@@ -50,7 +50,6 @@ stdenv.mkDerivation rec {
     libxslt # for xsltproc for building docs
     gobject-introspection
     wrapGAppsNoGuiHook
-    python3.pkgs.wrapPython
     # checkInput but cheked for during the configuring
     (python3.pythonOnBuildForHost.withPackages (ps: with ps; [
       pygobject3
@@ -67,16 +66,14 @@ stdenv.mkDerivation rec {
     upower
     glib
     polkit
-    python3 # for cli tool
+    # for cli tool
+    (python3.withPackages (ps: [
+      ps.pygobject3
+    ]))
   ];
 
   strictDeps = true;
 
-  # for cli tool
-  pythonPath = [
-    python3.pkgs.pygobject3
-  ];
-
   checkInputs = [
     umockdev
   ];
@@ -102,7 +99,9 @@ stdenv.mkDerivation rec {
   postPatch = ''
     patchShebangs --build \
       tests/integration-test.py \
-      tests/unittest_inspector.py \
+      tests/unittest_inspector.py
+
+    patchShebangs --host \
       src/powerprofilesctl
   '';
 
```

Reply:
Oops, indeed, makes sense, thanks!